### PR TITLE
add two datetime and number of person selection

### PR DESCRIPTION
**Pull request by gyusss**
add two datetime and number of person selection

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "react": "16.5.0",
     "react-native": "https://github.com/expo/react-native/archive/sdk-32.0.0.tar.gz",
     "react-native-modal-datetime-picker": "^7.4.0",
+    "react-native-modal-selector": "^1.0.3",
```
Comment (apple12347678) [inline package.json:22]: 더 좋은 패키지 찾아봅시다...

File: screens/PlanSecondScreen.js [hunk 2]
```diff
@@ -21,48 +22,73 @@ const styles = {
...
+            mode="datetime"
+            isVisible={state.isEndDateTimePickerVisible}
+            onConfirm={this._handleEndDatePicked}
+            onCancel={this._hideEndDateTimePicker}
+          />
+          <ModalSelector
+            data={data}
+            initValue="인원수 선택"
+            onChange={option => this.setState({ textInputValue: option.label })}
+          />
```
Comment (apple12347678) [inline screens/PlanSecondScreen.js:140]: 컨펌 버튼 만들어서 누르면 navigation.navigate('PlanConfirm', { startDate, endDate, origin, destination, maxMembers }) 실행되도록 짜주세요.

File: package.json (hunk 1 above)
Comment (apple12347678) [reply package.json:22]: https://www.npmjs.com/package/react-native-picker-select

**Review by jungdj — APPROVED** [review]
APPROVE

**Pull request merged**